File: MPI/mpicheck/tests/functions/4.0/MPI_Pack_external_size_c.f

        
        program main
        include 'mpif.h'
        
        CHARACTER*(10) var_0
       INTEGER(KIND=MPI_COUNT_KIND) var_1
       INTEGER var_2
       INTEGER(KIND=MPI_COUNT_KIND) var_3
       INTEGER var_4
        call mpi_pack_external_size_c(var_0, var_1, var_2, var_3, var_4)
       call pmpi_pack_external_size_c(var_0, var_1, var_2, var_3, var_4)
        end program main
    

File: MPI/mpicheck/tests/functions/4.0/MPI_Pack_external_size_c.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    const char var_0[2];
    MPI_Count var_1;
    MPI_Datatype var_2;
    MPI_Count *var_3;
    int ret;
    /* calls */
    ret = MPI_Pack_external_size_c(var_0, var_1, var_2, var_3);
    ret = PMPI_Pack_external_size_c(var_0, var_1, var_2, var_3);
    return 0;
}
